README.md

```diff
@@ -444,6 +445,24 @@ Note: For modules use the [@import](http://clang.llvm.org/docs/Modules.html#usin
 #import "NYTUserView.h"
 ```
 
+## Protocols
+
+In a [delegate or data source protocol](https://developer.apple.com/library/ios/documentation/General/Conceptual/CocoaEncyclopedia/DelegatesandDataSources/DelegatesandDataSources.html), the first parameter to each method should be the object sending the message.
+
+This aids in disambiguating in case an object is the delegate for multiple similarly-typed objects, and it helps clarify intent to readers of a class implementing these delegate methods.
```

Comment:
"This aids in disambiguating in case an object…" reads a little strangely. I’d suggest something like “This aids in disambiguating cases in when an object…”